ABB.SrcML.Data.Test/CodeParserTests.cs

```diff
@@ -44,10 +47,13 @@ public class CodeParserTests {
         [TestCase(Language.CPlusPlus)]
         [TestCase(Language.CSharp)]
         [TestCase(Language.Java)]
-        public void TestTwoVariableDeclarations(Language lang) {
-            //int a,b;
-            string testXml = @"<decl_stmt><decl><type><name>int</name></type> <name>a</name></decl><op:operator>,</op:operator><decl><type ref=""prev""/><name>b</name></decl>;</decl_stmt>";
-            var testUnit = fileSetup[lang].GetFileUnitForXmlSnippet(testXml, "test.cpp");
+        public void TestTwoVariableDeclarations(Language lang)
+        {
+            int a, b;
```

Comment:
This line should be removed. It was actually a comment before, but is unneeded now.